app/models/csv_files/csv_file.rb

```diff
@@ -0,0 +1,147 @@
+class CsvFile < ActiveRecord::Base
+  attr_accessor :upload
+
+  STI = %W(WeamsCsvFile)
```

Comment:
What is STI? 

This also looks like a list that appeared elsewhere in another module. If this is the case, can it be centralized?